x/participationrewards/keeper/distribution.go

```diff
@@ -209,3 +206,45 @@ func (k *Keeper) DistributeToUsers(ctx sdk.Context, userAllocations []types.User
 
 	return nil
 }
+
+// DistributeToUsers sends the allocated user rewards to the user address.
+func (k *Keeper) DistributeToUsersFromAddress(ctx sdk.Context, userAllocations []types.UserAllocation, fromAddress string) error {
+	k.Logger(ctx).Info("distributeToUsers", "allocations", userAllocations)
+	hasError := false
+
+	fromAddrBytes, err := addressutils.AccAddressFromBech32(fromAddress, "")
+	if err != nil {
+		k.Logger(ctx).Error("unmarshalling address", "address", fromAddress)
+		hasError = true
+	}
```

Comment:
If there is an error here, shouldnt we just return now?  If there is an error getting the bytes, won't there definitely be errors later?

Reply:
Yup, will remove the whole hasError thing from this and the similar function it was CP from :)